diff --git a/src/KDDockWidgetsConfig.cmake.in b/src/KDDockWidgetsConfig.cmake.in
index fbec60afb..31d45e5cd 100644
--- a/src/KDDockWidgetsConfig.cmake.in
+++ b/src/KDDockWidgetsConfig.cmake.in
@@ -24,5 +24,8 @@ if (NOT WIN32 AND NOT APPLE AND NOT EMSCRIPTEN AND NOT @KDDockWidgets_QT6@ AND @
     find_dependency(Qt5X11Extras REQUIRED)
 endif()
 
+find_dependency(KDBindings CONFIG)
+find_dependency(nlohmann_json)
+
 # Add the targets file
 include("${CMAKE_CURRENT_LIST_DIR}/KDDockWidgets@KDDockWidgets_LIBRARY_QTID@Targets.cmake")
diff --git a/src/kdbindings.cmake b/src/kdbindings.cmake
index e8af8acc1..977e301e8 100644
--- a/src/kdbindings.cmake
+++ b/src/kdbindings.cmake
@@ -15,6 +15,7 @@ function(kddw_link_to_kdbindings target)
     if(TARGET KDAB::KDBindings)
         target_link_libraries(${target} PRIVATE KDAB::KDBindings)
     else()
-        target_include_directories(${target} SYSTEM PRIVATE ${KDDW_3DPARTY_DIR})
+	find_package(KDBindings CONFIG REQUIRED GLOBAL)
+	add_library(kdbindings ALIAS KDAB::KDBindings)
     endif()
 endfunction()
