diff --git a/CMakeLists.txt b/CMakeLists.txt
index f15a3ec..079e6dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -302,21 +302,21 @@ endif()
 ### Install  ###
 if(NOT LIBREMIDI_HEADER_ONLY)
   install(TARGETS libremidi
-          EXPORT libremidi-targets
+          EXPORT libremidi-config
           ARCHIVE DESTINATION lib/static
           RUNTIME DESTINATION bin
           LIBRARY DESTINATION lib
   )
 else()
   install(TARGETS libremidi
-          EXPORT libremidi-targets
+          EXPORT libremidi-config
   )
 endif()
-install(EXPORT libremidi-targets
+install(EXPORT libremidi-config
         DESTINATION lib/cmake/libremidi)
 install(DIRECTORY include
         DESTINATION .)
-export(EXPORT libremidi-targets)
+export(EXPORT libremidi-config)
 
 include(CMakePackageConfigHelpers)
 
