diff --git a/CMakeLists.txt b/CMakeLists.txt
index b2ef949..3e1a304 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,6 +54,21 @@ endif()
 
 add_library(reckless STATIC ${SRC_LIST})
 
+install(TARGETS reckless
+  EXPORT recklessTargets
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  INCLUDES DESTINATION include
+)
+
+install(DIRECTORY reckless/include/
+  DESTINATION include)
+
+install(EXPORT recklessTargets
+  FILE recklessConfig.cmake
+  NAMESPACE reckless::
+  DESTINATION lib/cmake/reckless)
+
 ################################################################################
 # Build Examples
 ################################################################################
