diff --git a/imgui.cpp b/imgui.cpp
index b5fa1bf2..5219e3a7 100644
--- a/imgui.cpp
+++ b/imgui.cpp
@@ -1200,7 +1200,8 @@ ImGuiIO::ImGuiIO()
     for (int i = 0; i < IM_ARRAYSIZE(MouseDownDuration); i++) MouseDownDuration[i] = MouseDownDurationPrev[i] = -1.0f;
     for (int i = 0; i < IM_ARRAYSIZE(KeysData); i++) { KeysData[i].DownDuration = KeysData[i].DownDurationPrev = -1.0f; }
     for (int i = 0; i < IM_ARRAYSIZE(NavInputsDownDuration); i++) NavInputsDownDuration[i] = -1.0f;
-    BackendUsingLegacyKeyArrays = (ImS8)-1;
+    /* BackendUsingLegacyKeyArrays = (ImS8)-1; */
+    BackendUsingLegacyKeyArrays = (ImS8)1;
     BackendUsingLegacyNavInputArray = true; // assume using legacy array until proven wrong
 }
 
